.ci/common/test.sh

```diff
@@ -110,7 +110,16 @@ run_oldtests() {
 }
 
 run_single_includes_tests() {
-  ${MAKE_CMD} -C "${BULID_DIR}" check-single-includes
+  ulimit -c unlimited
+  if ! ${MAKE_CMD} -C "${BUILD_DIR}" check-single-includes; then
+    asan_check "${LOG_DIR}"
+    valgrind_check "${LOG_DIR}"
+    check_core_dumps
```

Comment:
These are useless: test only checks whether some things *compile*. It does not even run anything, and even if it did there is no reason for these executables to trigger any of the above errors: they are mere `int main() { return 0; }` with a few dead codes (`static inline` functions in headers) and a big bunch of various defines (typedefs, macros, globals).

You can theoretically find a bug in compiler or CMake via `check_core_dumps`, but `check_core_dumps` is not the right function for this task (it assumes a single app, normally Neovim or luajit). `asan_check` and `valgrind_check` are simply useless, nobody bothered to run CMake or compilers under valgrind or compile them with sanitizers enabled.